README.md

```diff
@@ -168,11 +168,10 @@ const signer = new ethers.Wallet(privateKey, provider);
 const offchainAttestation = await offchain.signOffchainAttestation({
   recipient: '0xFD50b031E778fAb33DfD2Fc3Ca66a1EeF0652165',
   // Unix timestamp of when attestation expires. (0 for no expiration)
-  expirationTime: 0,
+  expirationTime: BigInt(0),
   // Unix timestamp of current time
-  time: 1671219636,
+  time: BigInt(Math.floor(Date.now() / 1000)),
```

Comment:
Thanks for the contribution! 

It'd be more idiosyncratic to use `0n` instead of `BigInt(0)`. Could you update it?

Reply:
Good point! Updated along with some other suggestions based on our v0 implementation yesterday.

Reply:
Thanks! Could you also update `nonce` to `0n` as well (and then it's good to go)?

Reply:
Ah, thanks for catching. We removed that variable from our own implementation so didn't notice. Fixed.